release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -186,6 +186,13 @@ For more information, see xref:../web_console/dynamic-plugin/dynamic-plugins-ref
 
 For more information about `console.resource/details-item`, see xref:../web_console/dynamic-plugin/dynamic-plugins-reference.adoc#dynamic-plugin-sdk-extensions_dynamic-plugins-reference[{product-title} console API].
 
+[id="ocp-4-15-console-supports-azure-sts-detection"]
+===== OperatorHub support for {azure-id}
+
+With this release, OperatorHub detects when a {product-title} cluster running on Azure is configured for {azure-id}. When detected, a "Cluster in Workload Identity / Federated Identity Mode" notification displays with additional instructions before installing an Operator to ensure it runs correctly. The *Operator Installation* page is also modified to add fields for the required Azure credentials information.
```

Comment:
```suggestion
With this release, OperatorHub detects when a {product-title} cluster running on Azure is configured for {azure-id}. When detected, a "Cluster in Workload Identity / Federated Identity Mode" notification is displayed with additional instructions before installing an Operator to ensure it runs correctly. The *Operator Installation* page is also modified to add fields for the required Azure credentials information.
```
I went back and forth on this one, but ISG says "[Use only](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d) as a transitive verb. For example, write “the system displays a message” or “the message is displayed” (not “the message displays”)" So I am wondering if this falls under what they have in parenthesis.  I think "is displayed" might work better here but honestly the way the style guide has it it seems both are correct, so up to you!